### DISCUSSION TITLE
Add custom "Favorites" side tabs ?

### DISCUSSION THREAD

**Discussion by leodom** [elementary/files#2363] [Ideas]
Hello !
I do bookmark quite a few folders and sftp connections.
I was wondering if there is an easy way to create fold up menus like "Favorites/Storage/Network"
so I can tidy up my bookmarks and sites ?
Bookmarks subfolders would be a dream, though :)
Thanks for pointers...

**Comment by jeremypw** [comment]
Files does not implement the ability to add additional bookmark groups.  These are not catered for in the freedesktop specification at present I believe. We could do this by implementing our own bookmark storage specification (as e.g. Chrome does) but then the bookmarks would not be readable by other applications.    A work round you could try would be to create a folder called e.g. "My Bookmarks" and bookmark this in the sidebar. You can then put links to the desired folders and files into this folder.  Do this by right-clicking on the file you and choose "Copy as Link". Then navigate to the "My Bookmarks" folder, right-click, and choose "Paste Link".  The items in this folder will then act like bookmarks.

**Reply by leodom** [reply]
Thanks a lot for your reply Jeremy, 

it is quite a good solution for local files and folders,
but sadly I can't do it with remote folders, receiving
a message saying symlinks is only for local archives.

At least I can space up for the remote servers bookmarks.

Out of curiosity, any simple files manager you'd know 
with bookmarks subfolder ability ?

**Reply by jeremypw** [reply]
I've not found one. CoreFM seemed to have an "Add Section" option in the "Pin It" function but it didn't seem to work on elementary.  Maybe only works on KDE.

**Comment by jeremypw** [comment]
I'll convert this to an issue but the priority at the moment is to port to Gtk4, not add new functionality.
